druid/src/main/java/org/apache/calcite/adapter/druid/TimeExtractionFunction.java

```diff
@@ -94,7 +117,7 @@ public static TimeExtractionFunction createDefault(String timeZone) {
    */
   public static TimeExtractionFunction createExtractFromGranularity(
       Granularity granularity, String timeZone) {
-    final String local = Locale.ROOT.toLanguageTag();
+    final String local = Locale.US.toLanguageTag();
```

Comment:
Why did we change locale object for US here?

Reply:
it is safer to have a fixed one, in fact, it is not really needed can be left as null since we are building the format string anyway.